orchestrator_service/src/main/java/com/override/orchestrator_service/controller/rest/TransactionController.java

```diff
@@ -65,58 +74,75 @@ public List<TransactionDTO> getTransactionsList(Principal principal) throws Inst
     }
 
     @GetMapping("/transactions/info")
-    public List<TransactionDTO> getTransactionsListByPeriodAndCategory(@RequestParam Integer year,
-                                                                       @RequestParam Integer month,
-                                                                       @RequestParam long categoryId) {
+    @Operation(summary = "Получить список транзакций по периоду и категории", description = "Возвращает список транзакций за указанный период")
+    public List<TransactionDTO> getTransactionsListByPeriodAndCategory(
+            @Parameter(description = "Год транзакции") @RequestParam Integer year,
+            @Parameter(description = "Месяц транзакции") @RequestParam Integer month,
+            @Parameter(description = "ID категории транзакции") @RequestParam long categoryId) {
         return transactionService.getTransactionsListByPeriodAndCategory(year, month, categoryId);
     }
 
     @GetMapping("/transactions/history")
+    @Operation(summary = "Получить историю транзакций", description = "Возвращает историю транзакций")
     public List<TransactionDTO> getTransactionsHistory(Principal principal,
-                                                       @RequestParam(defaultValue = "50") Integer pageSize,
-                                                       @RequestParam(defaultValue = "0") Integer pageNumber)
+                                                       @Parameter(description = "Размер страницы") @RequestParam(defaultValue = "50") Integer pageSize,
```

Comment:
```suggestion
                                                       @Parameter(description = "Транзакций на странице") @RequestParam(defaultValue = "50") Integer pageSize,
```

размер страницы не понятно что такое